x/mint/types/minter.go

```diff
@@ -19,17 +17,19 @@ func NewMinter(inflation, annualProvisions math.LegacyDec) Minter {
 
 // InitialMinter returns an initial Minter object with a given inflation value.
 func InitialMinter(inflation math.LegacyDec) Minter {
+
 	return NewMinter(
 		inflation,
-		math.LegacyNewDec(0),
+		// 2831000000000000000000 (initial uallo per block) * 6311520 (blocks per year) = 17867913120000000000000000 (initial annual provisions)
+		math.LegacyNewDecFromBigInt(math.NewUintFromString("17867913120000000000000000").BigInt()),
 	)
 }
 
 // DefaultInitialMinter returns a default initial Minter object for a new chain
-// which uses an inflation rate of 13%.
+// which uses an inflation rate of 15,03%.
 func DefaultInitialMinter() Minter {
 	return InitialMinter(
-		math.LegacyNewDecWithPrec(13, 2),
+		math.LegacyNewDecWithPrec(3573582624, 7),
```

Comment:
is this number right lol?

Reply:
It's the calculated initial inflation. Inflation now it's just used for query purposes, it will be updated every block.